epel/map.jinja

```diff
@@ -3,12 +3,22 @@
       'disabled': False,
       'testing': False,
     },
-    'Amazon': {
-      'key': 'https://fedoraproject.org/static/0608B895.txt',
-      'key_hash': 'sha256=16d35fa467c6efcee21d3aa068a02054b6a89a7a33bffa94db1fc141693d62a3',
-      'key_name': 'RPM-GPG-KEY-EPEL-6',
-      'rpm': 'http://download.fedoraproject.org/pub/epel/6/i386/epel-release-6-8.noarch.rpm',
+    'Amazon': salt['grains.filter_by']({
+      '2': {
+        'key': 'https://fedoraproject.org/static/352C64E5.txt',
+        'key_hash': 'sha256=22f25ad95d5e8d371760815485dba696ea3002fc2c7f812f2c75276853387107',
+        'key_name': 'RPM-GPG-KEY-EPEL-7',
+        'rpm': 'https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm',
+      },
+      '2018': {
+        'key': 'https://fedoraproject.org/static/0608B895.txt',
+        'key_hash': 'sha256=16d35fa467c6efcee21d3aa068a02054b6a89a7a33bffa94db1fc141693d62a3',
+        'key_name': 'RPM-GPG-KEY-EPEL-6',
+        'rpm': 'http://download.fedoraproject.org/pub/epel/6/i386/epel-release-6-8.noarch.rpm',
+      },
     },
+    grain='osmajorreleaes',
```

Comment:
```suggestion
    grain='osmajorrelease',
```

@mew1033 Since this has already been merged, would you mind putting this in another PR?